src/common/BlogApi.res

```diff
@@ -109,21 +115,16 @@ module RssFeed = {
   let getLatest = (~max=10, ~baseUrl="https://rescript-lang.org", ()): array<item> => {
     let items =
       getAllPosts()
-      ->Belt.Array.reduce([], (acc, next) =>
-        switch BlogFrontmatter.decode(next.frontmatter) {
-        | Ok(fm) =>
-          let description = Js.Null.toOption(fm.description)->Belt.Option.getWithDefault("")
-          let item = {
-            title: fm.title,
-            href: baseUrl ++ "/blog/" ++ blogPathToSlug(next.path),
-            description: description,
-            pubDate: DateStr.toDate(fm.date),
-          }
-
-          Belt.Array.concat(acc, [item])
-        | Error(_) => acc
```

Comment:
Why use `result` if we don't handle the error.

Good thing we're not covering it up with some extra let-ppx too.